src/components/pages/confirm/membership/index.tsx

```diff
@@ -39,10 +44,12 @@ const PrimaryMessage: React.FC = ({children}) => {
   return <div className="dark:text-gray-200 text-gray-800">{children}</div>
 }
 
+const tweet = `https://twitter.com/intent/tweet/?text=Just joined @eggheadio to level up my development skills.`
```

Comment:
I added a little cta to tweet about egghead. @dealingwith copy suggestions welcome!